CONTRIBUTING.md

```diff
@@ -221,6 +221,22 @@ Integration tests are not run by default when running
 `cargo vdev test`. Instead, they are accessible via the integration subcommand (example:
 `cargo vdev int test aws` runs aws-related integration tests). You can find the list of available integration tests using `cargo vdev int show`. Integration tests require docker or podman to run.
 
+### Running other checks
+
+There are other checks that are run by CI before the PR can be merged. These should be run locally
+first to ensure they pass.
+
+- `cargo vdev check rust` - this runs the Clippy linter to catch common mistakes.
```

Comment:
This is looks like a great addition to me. But we can make this copy-pastable:
 ```bash
cargo vdev check rust # this runs the Clippy linter to catch common mistakes.
cargo vdev check fmt # this ensures all code is properly formatted. Code can be run through
# ...
 ````